### PR TITLE
feat: errors inbox quickstart | CircleCI quickstart

### PR DESCRIPTION
**Pull request by jcountsNR**
# Summary

New documentation only quickstart. No dashboard or alerts included and no install plan.

## Pre merge checklist

- [ X] Did you check your descriptive content for voice, tone, spelling and grammar errors?

### CODE REVIEW

File: quickstarts/jira-errors/config.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Name of the quickstart (required)
+name: jira-errors-inbox
+
+# Displayed in the UI (required)
+title: Atlassian Jira for Errors Inbox
+
+# Long-form description of the quickstart (required)
+description: |
+  [Connect errors inbox to Jira (Cloud)](https://newrelic.com/blog/how-to-relic/errors-inbox-with-jira) to easily create tickets for your errors, allowing for faster collaboration and resolution. Check out our documentation to learn how to set up Jira templates, which allow you to quickly create a ticket containing error details and links directly to the stack trace and APM for quick access and resolution.
```
Comment (jpvajda) [inline quickstarts/jira-errors/config.yml:9]: Is there a link to to a docs site page we can reference here vs a blog link?

File: quickstarts/jira-errors/config.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - inbox
+  - newrelic
+  - atlassian
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+
+documentation:
+  - name: How to Docs
+    url: https://newrelic.com/blog/how-to-relic/errors-inbox-with-jira
```
Comment (jpvajda) [inline quickstarts/jira-errors/config.yml:36]: Is there a link to to a docs site page we can reference here vs a blog link?

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: quickstarts/jira-errors/config.yml (hunk 2 above)
Comment (jcountsNR) [reply quickstarts/jira-errors/config.yml]: done.

**Comment by jpvajda** [comment]
@jcountsNR Thank you! In the future can you ensure a PR focuses on a single quickstart addition? I know it's easier to try to put it all into one PR, but we do like to keep the PRs seperate for new quickstarts ideally.

File: install/third-party/circleci/install.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+id: third-party-circleci
+name: CircleCi
+title: CircleCI quickstart
+description:
+  Provides views to track the performance and health of your continuous integration and deployment pipelines.
```
Comment (jpvajda) [inline install/third-party/circleci/install.yml:5]: I'd recommend a more verbose description here. this is pretty sparse.

Comment (jpvajda) [reply install/third-party/circleci/install.yml:5]: actually this is the install.yml description, so never mind. 😄

File: quickstarts/jumpstart/config.yml [hunk 4]
```diff
@@ -22,9 +22,11 @@ authors:
...
 # Keywords for filtering / searching criteria in the UI
 keywords:
-  - Jumpstart
-  - Quickstart
-  - Intro
+  - jumpstart
+  - quickstart
+  - intro
+  - how-to
+  - create
```
Comment (jpvajda) [inline quickstarts/jumpstart/config.yml:29]:
```suggestion
```

File: quickstarts/jumpstart/config.yml [hunk 5]
```diff
@@ -22,9 +22,11 @@ authors:
 
 # Keywords for filtering / searching criteria in the UI
 keywords:
-  - Jumpstart
-  - Quickstart
-  - Intro
+  - jumpstart
+  - quickstart
+  - intro
+  - how-to
```
Comment (jpvajda) [inline quickstarts/jumpstart/config.yml:28]:
```suggestion
```

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: quickstarts/jumpstart/config.yml (hunk 4 above)
Comment (jpvajda) [reply quickstarts/jumpstart/config.yml:29]: this feels unnecessary as a keyword so I suggest we remove it.

File: quickstarts/jumpstart/config.yml (hunk 5 above)
Comment (jpvajda) [reply quickstarts/jumpstart/config.yml:28]: this feels unnecessary as a keyword so I suggest we remove it.

Comment (jcountsNR) [reply quickstarts/jumpstart/config.yml:28]: Sure, the main thing was to fix the case since the querying is case sensitive.

**Comment by jcountsNR** [comment]
> @jcountsNR Thank you! In the future can you ensure a PR focuses on a single quickstart addition? I know it's easier to try to put it all into one PR, but we do like to keep the PRs separate for new quickstarts ideally.

got it, I only had one branch so when I committed it went straight into this PR. I'll create a separate branch next time.

**Review by jpvajda — APPROVED** [review]

**Pull request merged**